src/core/server/http/http_server.ts

```diff
@@ -465,7 +465,13 @@ export class HttpServer {
           lookupCompressed: true,
         },
       },
-      options: { auth: false },
+      options: {
+        auth: false,
+        cache: {
+          privacy: 'public',
+          otherwise: 'must-revalidate',
```

Comment:
AFAIK, we're forced to specify an `otherwise` option to override the global value specified when instantiating the server. I would have expected another name for that, but eh, it's working.

Reply:
Maybe it's worth adding this comment to the route declaration?